semialign/src/Data/Crosswalk.hs

```diff
@@ -55,15 +62,15 @@ instance Crosswalk [] where
     crosswalk f (x:xs) = alignWith cons (f x) (crosswalk f xs)
       where cons = these pure id (:)
 
+instance Crosswalk NE.NonEmpty where
+    crosswalk f (x NE.:| []) = (NE.:| []) <$> f x
+    crosswalk f (x1 NE.:| x2 : xs) = alignWith cons (f x1) (crosswalk f (x2 NE.:| xs))
```

Comment:
can't this take advantage of `Crosswalk []` instance (c.f. `Traversable NonEmpty` instance)

Reply:
It could. The advantage of this implementation is that it only requires `Semialign` (doesnt use `nil`)

Reply:
I see, that's good.